ethernet-view/src/components/ChartMenu/ChartList/ChartList.tsx

```diff
@@ -1,51 +1,39 @@
 import styles from "./ChartList.module.scss";
-import { ChartWithLegend } from "./ChartWithLegend/ChartWithLegend";
+import { ChartElement } from "./ChartElement/ChartElement";
 import { DragEvent, useCallback } from "react";
 import { useCharts } from "../useCharts";
+import { MeasurementInfo } from "../types";
 import { nanoid } from "nanoid";
-import { ChartLine } from "../ChartElement";
 
 type Props = {
-    getLine: (id: string) => ChartLine;
+    getMeasurementInfo: (id: string) => MeasurementInfo;
 };
 
-export const ChartList = ({ getLine }: Props) => {
-    const { charts, addChart, removeChart, addLine, removeLine } = useCharts();
+export const ChartList = ({ getMeasurementInfo }: Props) => {
+    const { charts, addChart, removeChart, addMeasurementToChart, removeMeasurementFromChart } = useCharts();
 
     const handleDrop = useCallback((ev: DragEvent<HTMLDivElement>) => {
```

Comment:
for a previous comment, here you have the type instead of `any`

Reply:
I don't know why, but Typescript throws an error when I specify the other event type, not with this one. I have to take a look at it.